src/node.cc

```diff
@@ -3616,6 +3617,8 @@ static void PrintHelp() {
          "  --replay-debug=dir         replay and debug using recording log\n"
          "  --break-first              break at first statement when running\n"
          "                             in --replay-debug mode\n"
+         "  --disableAutoTrace       disable the auto-trace feature during\n"
```

Comment:
Is there a tabs/spaces issue here? The explanation text doesn't seem to be aligned with the other options.

Reply:
Not sure but fixed and updated PR.